tools/p2p_ansible_test/run_p2ptest.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Define variables
+TALKER_DEST="/home/kamber/ans_talker"
```

Comment:
Add comments describing each variable, so the users can customize the script.

I am not sure if it can be used here, but perhaps you can use temp directories instead, that are created from the ansible playbook itself. So the users would not need to customize the script. I am creating a follow-up ticket to evaluate that.